apps/end-tenancy/controllers/confirm.js

```diff
@@ -48,14 +48,17 @@ module.exports = class ConfirmController extends controllers.confirm {
     const loopStep = this.options.steps['/tenant-details'];
     const tenants = req.sessionModel.get(loopStep.storeKey);
 
+    const getSubStep = (field, subSteps) =>
+      _.findKey(subSteps, subStep => subStep.fields.indexOf(field) > -1);
+
     const fields = _.flatten(
       _.map(tenants, (tenant, id) =>
         _.map(
           _.pickBy(tenant, (value, field) =>
             this.options.fieldsConfig[field].includeInSummary !== false), (value, field) => ({
```

Comment:
yes, `this` is that of the parent, which in this case is another arrow block, and anothr, and another, and finally the controller!